sympy/physics/vector/dyadic.py

```diff
@@ -1,4 +1,8 @@
-from sympy import sympify, Add, ImmutableMatrix as Matrix
+from sympy.core.backend import sympify, AddClass as Add, USE_SYMENGINE
+if USE_SYMENGINE:
+    from sympy.core.backend import Matrix
+else:
+    from sympy import ImmutableMatrix as Matrix
```

Comment:
I don't have an idea why ImmutableMatrix is used, but I'm going to implement a ImmutableMatrix type in symengine.py (not symengine). This is a temporary fix. symengine/symengine.py#76

I didn't want to change anything in SymPy, that's why sympy version wasn't changed to a mutable matrix.